cmd/vclusterctl/cmd/create.go

```diff
@@ -18,13 +18,14 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/wait"
 
+	"github.com/loft-sh/utils/pkg/helm/values"
```

Comment:
there are a few more functions like that `GetKubernetesVersion` and `GetKubernetesMinorVersion` but I'm not sure if we need another package for it right now. But it could be done in another PR if that is required.